streams/src/main/java/org/apache/kafka/streams/state/internals/MeteredKeyValueStore.java

```diff
@@ -253,12 +262,17 @@ public boolean setFlushListener(final CacheFlushListener<K, V> listener,
             rawRangeQuery = RangeQuery.withNoBounds();
         }
         final QueryResult<KeyValueIterator<Bytes, byte[]>> rawResult =
-                wrapped().query(rawRangeQuery, positionBound, collectExecutionInfo);
+            wrapped().query(rawRangeQuery, positionBound, collectExecutionInfo);
         if (rawResult.isSuccess()) {
             final KeyValueIterator<Bytes, byte[]> iterator = rawResult.getResult();
             final KeyValueIterator<K, V> resultIterator = new MeteredKeyValueTimestampedIterator(
-                    iterator, getSensor, getValueDeserializer());
-            final QueryResult<KeyValueIterator<K, V>> typedQueryResult = QueryResult.forResult(resultIterator);
+                iterator,
+                getSensor,
+                getValueDeserializer()
+            );
+            final QueryResult<KeyValueIterator<K, V>> typedQueryResult = rawResult.swapResult(
+                resultIterator
```

Comment:
nit: why newline with just one parameter?

Reply:
Probably autoformatted because the line was too long.